fotoapparat/src/main/java/io/fotoapparat/hardware/v1/Camera1.java

```diff
@@ -433,4 +443,26 @@ private void recordMethod() {
         );
     }
 
+    @Override
+    public void setFocusArea(Rect cameraViewRect, float x, float y) {
+        Camera.CameraInfo info = getCameraInfo(cameraId);
+        SpaceConverter spaceConverter = new SpaceConverter(
+                displayOrientationDegrees,
+                info.facing == Camera.CameraInfo.CAMERA_FACING_FRONT
+        );
+
+        Rect previewSpaceRect = spaceConverter.previewToCamera(cameraViewRect, x, y);
+
+        List<Camera.Area> focusingAreas = new ArrayList<>();
+        focusingAreas.add(new Camera.Area(previewSpaceRect, FOCUS_METERING_AREA_WEIGHT_DEFAULT));
+
+        FocusUpdater focusUpdater = new FocusUpdater(
+                CameraParametersDecorator.from(camera),
+                getCapabilities()
+        );
+
+        if (focusUpdater.setManualFocus(focusingAreas)) {
```

Comment:
Setters should do one thing which means it should not return `boolean`. I would instead write it like that:

```java
if (focusUpdater.shouldUpdateManualFocus(getCapabilities())) {
    camera.setParameters(focusUpdater.buildParameters(focusingAreas).asCameraParameters());
}
```